programs/ssl/ssl_context_info.c

```diff
@@ -299,10 +301,11 @@ void print_hex( const uint8_t *b, size_t len,
 /*
  *  Print the value of time_t in format e.g. 2020-01-23 13:05:59
  */
-void print_time( const time_t *time )
+void print_time( const uint64_t *time )
 {
+#if defined(MBEDTLS_HAVE_TIME)
     char buf[20];
-    struct tm *t = gmtime( time );
+    struct tm *t = gmtime( (time_t*)time );
```

Comment:
You'll hate me for this, but I have a horrible feeling the coding style would prefer

```
struct tm *t = gmtime( (time_t *) time );
```

Reply:
All good :)